### PR TITLE
WIP: Quarkus SDK

### PR DESCRIPTION
**Pull request by paulcastro**
This is a WIP of a REST client built natively for Quarkus.  It's mainly skeleton code and build files to get started.

- Quarkus application to drive simple KAR call to the server for `hello-java`
- Implementation of service call using Vert.x WebClient.
- framework to configure HTTP/2 client (issue #46)

Usage: in `sdk-quarkus/kar-rest/client` do `kar run -h2c -v info -app hello-java ./mvnw compile quarkus:dev`

### CODE REVIEW

File: sdk-quarkus/kar-rest-client/.mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
+/*
```
Comment (dgrove-oss) [inline sdk-quarkus/kar-rest-client/.mvn/wrapper/MavenWrapperDownloader.java:1]: I suspect the entire .mvn subdirectory should not be committed to git.

File: sdk-quarkus/kar-rest-client/mvnw [hunk 2]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```
Comment (dgrove-oss) [inline sdk-quarkus/kar-rest-client/mvnw:1]: I know quarkus generates this in its sample project, but I think we don't actually need it (we pre-req maven being installed for other parts of kar)

File: sdk-quarkus/kar-rest-client/mvnw.cmd [hunk 3]
```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```
Comment (dgrove-oss) [inline sdk-quarkus/kar-rest-client/mvnw.cmd:1]: Same comment here; shouldn't need to commit this file

**Review by dgrove-oss — CHANGES_REQUESTED** [review]

File: sdk-quarkus/kar-rest-client/.mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (paulcastro) [reply sdk-quarkus/kar-rest-client/.mvn/wrapper/MavenWrapperDownloader.java]: Makes sense, I'll delete the generated maven artifacts.

**Comment by paulcastro** [comment]
@dgrove-oss , I removed the superfluous generated artifacts

**Comment by paulcastro** [comment]
Yikes, looks like removing the maven wrappers breaks the build on my dev machine. I'll re-open this once I figure out what's wrong.

**Pull request closed without merging**